perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/scheduling/impl/SchedulingPoolImpl.java

```diff
@@ -69,19 +68,18 @@
 	private DelayQueue<TaskSchedule> waitingForcedTasksQueue;
 	private Properties dispatcherProperties;
 	private TaskStore taskStore;
-	private TaskManager taskManager;
-	private ResultManager resultManager;
+	private TasksManagerBl tasksManagerBl;
```

Comment:
Since there is ``perun`` bean already present, you can use its ``((PerunBl)perun).getTasksManagerBl()`` to get ``TasksManagerBl`` instance.